test/validation/api/pool/pool.c

```diff
@@ -1175,11 +1534,26 @@ odp_testinfo_t pool_suite[] = {
 	ODP_TEST_INFO_NULL,
 };
 
+odp_testinfo_t pool_ext_suite[] = {
+	ODP_TEST_INFO(test_packet_pool_ext_capa),
+	ODP_TEST_INFO(test_packet_pool_ext_create),
+	ODP_TEST_INFO(test_packet_pool_ext_lookup),
+	ODP_TEST_INFO(test_packet_pool_ext_info),
+	ODP_TEST_INFO(test_packet_pool_ext_populate),
+	ODP_TEST_INFO(test_packet_pool_ext_alloc),
+	ODP_TEST_INFO(test_packet_pool_ext_alloc_max),
+	ODP_TEST_INFO(test_packet_pool_ext_alloc_seg),
```

Comment:
External memory pool tests should be conditional.

Reply:
Fixed in v15. All test cases, except the first capa test, are conditional.